code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -93,13 +93,13 @@ \subsection{Table of Symbols}
 \\
 $interpZ$ & InterpZ & --
 \\
-$is-safeLoad$ & Variable that is assigned true when load resistance (capacity) is greater than applied load (demand) & --
+$is-safeLoad$ & Load resistance safety requirement & --
 \\
-$is-safeLR$ & Variable that is assigned true when load resistance (capacity) is greater than load (demand) & --
+$is-safeLR$ & Load resistance safety requirement & --
```

Comment:
The description of `is-safeLoad` and `is-safeLR` are the same.  We shouldn't really have symbols with the same definition.  My guess is that this due to the confusion of the general theory of safety and the specific instance of safety for GlassBR.  @samm82 I can't remember which is which, but please add wording that shows which one is the more general notion and which is specific to GlassBR.

Reply:
For the GlassBR specific quantity, how about "applied load resistance safety requirement", "demand resistance safety requirement", or "3 second load equivalent resistance safety requirement"? I'm trying to think of how to indicate that the load is the specific instance from GlassBR.

Reply:
@samm82, I like your last option "3 second ...".  Let us go with that one.  Thank you!